dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -33,13 +33,16 @@ cache.config=Authentication Cache
 cache.config.desc=Controls the operation of the authentication cache.
 
 initialSize=Initial cache size
-initialSize.desc=Initial number of entries supported by the authentication cache.
+initialSize.desc=Initial number of entries supported by the local authentication cache. This setting does not apply to the JCache.
```

Comment:
suggested edit

The initial number of entries that are supported by the local authentication cache. This setting does not apply to the JCache cache.

Reply:
Updated. Also changed "local" to "in-memory".